.travis.yml

```diff
@@ -1,15 +1,9 @@
 language: python
-
-python:
-  - "2.7"
-  - "3.5"
-
-install: 
-  - pip install nose
-  - python setup.py install
-
-before_script:
-  - cp config/config.example.yml config/config.yml
```

Comment:
This is an important step or the tests will fail, how is this handled in the new config?